open-api/rest-catalog-open-api.yaml

```diff
@@ -2494,6 +2495,28 @@ components:
           type: integer
           format: int64
 
+    AppendDataFileUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+      required:
+        - action
+        - added-data-files
+        - schema
+        - spec
+      properties:
+        action:
+          type: string
+          enum: [ "append-data-files" ]
+        added-data-files:
```

Comment:
should this be `appended-data-files`? to match the action name

Reply:
Or maybe just `data-files`?

Reply:
I'd +1 just `data-files`